### PR TITLE
refactor(Services): routes and services

### PR DESCRIPTION
**Pull request by aaron-goshine**
- Simplified routes
- Simplified Database connection
- Implemented services

### CODE REVIEW

**Comment by aaron-goshine** [comment]
@christopher-gordon98 @nic-ball @william-eyre @olikeys

File: services/users.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const router = express.Router();
+const env = require('dotenv').config();
+
+function getUser (username) {
+  const client = new Client({
+    connectionString: process.env.HEROKU_POSTGRESQL_PINK_URL
+  });
+
+  client.connect();
+  return client.query(`SELECT * FROM users WHERE username = '${username}'`);
```
Comment (CGDEV998) [inline services/users.js:13]: By returning the client query, does this then close the client connection? If not, should we not close the client connection before spinning up any new clients?

**Review by CGDEV998 — CHANGES_REQUESTED** [review]

File: services/users.js (hunk 1 above)
Comment (aaron-goshine) [reply services/users.js:13]: @christopher-gordon98 if feel confident go ahead and fix it

Comment (CGDEV998) [reply services/users.js:13]: I'm happy to merge without client disconnects, we're small scale enough that it won't matter for now

**Review by CGDEV998 — APPROVED** [review]
LGTM👍

**Pull request merged**